internal/runner/operation_init.go

```diff
@@ -0,0 +1,33 @@
+package runner
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-hclog"
+
+	"github.com/hashicorp/waypoint/internal/core"
+	pb "github.com/hashicorp/waypoint/internal/server/gen"
+)
+
+func (r *Runner) executeInitOp(
+	ctx context.Context,
+	log hclog.Logger,
+	project *core.Project,
+) (*pb.Job_Result, error) {
+	client := project.Client()
+
+	for _, name := range project.Apps() {
```

Comment:
I'd like to suggest adding a comment above here noting _why_ we're doing this.

Reply:
Added a comment. Let me know if it fits the bill.